pkg/autodiscovery/listeners/kube_endpoints.go

```diff
@@ -64,7 +66,7 @@ func init() {
 	Register(kubeEndpointsName, NewKubeEndpointsListener)
 }
 
-func NewKubeEndpointsListener(Config) (ServiceListener, error) {
+func NewKubeEndpointsListener(conf Config) (ServiceListener, error) {
```

Comment:
is `Config` an Interface?
if not, should we do instead

```suggestion
func NewKubeEndpointsListener(conf *Config) (ServiceListener, error) {
```

Reply:
I just check and indeed it is an interface 👍